### DISCUSSION TITLE
Wagtail 5.1 released

### DISCUSSION THREAD

**Discussion by laymonage** [wagtail/wagtail#10736] [Announcements]
Wagtail 5.1 is now available! Major new features in this release include:
- AVIF image support
- ElasticSearch 8 support
- Search query boosting for ElasticSearch 6+
- Custom template support for `wagtail start`
- Read-only `FieldPanel`s
- Performance improvements

And more. For the full details, see the release notes: https://docs.wagtail.org/en/stable/releases/5.1.html.

For general notes on upgrading, see https://docs.wagtail.org/en/stable/releases/upgrading.html.

**Comment by zerolab** [comment]
I would like to highlight the fact that in addition to AVIF support, you can now use image optimizer binaries (such as jpegoptim or pngquant) for smaller images overall - https://github.com/wagtail/Willow/releases/tag/v1.6
